lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -185,7 +185,7 @@ describe('CategoryRenderer', () => {
     const categoryDOM = renderer.render(category, sampleResults.categoryGroups);
     assert.ok(categoryDOM.querySelector('.lh-gauge__wrapper--plugin'));
     const label = categoryDOM.querySelector('.lh-gauge__label').textContent;
-    assert.equal(category.title, label);
+    assert.equal(label, category.title);
```

Comment:
was backwards

Reply:
> was backwards

just for code correctness purposes, not a functional change in the test, I hope?

Reply:
yes